packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -4,15 +4,43 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import React, { Component } from 'react';
-import { addressShort, Card, Form as FetherForm } from 'fether-ui';
+import BigNumber from 'bignumber.js';
+import { branch } from 'recompose';
+import { chainId$, chainName$ } from '@parity/light.js';
+import light from '@parity/light.js-react';
 import { inject, observer } from 'mobx-react';
+import { addressShort, Card, Form as FetherForm } from 'fether-ui';
 
 import RequireHealthOverlay from '../../../RequireHealthOverlay';
 import Scanner from '../../../Scanner';
 import withAccountsInfo from '../../../utils/withAccountsInfo';
+import withHealth from '../../../utils/withHealth';
+
+// Parity Signer networks that are available
+const PARITY_SIGNER_NETWORKS = {
+  1: 'foundation',
+  3: 'ropsten',
+  42: 'kovan',
+  61: 'classic'
+};
 
 @withAccountsInfo
+@withHealth
 @inject('createAccountStore')
+@light({
+  chainId: () => chainId$()
+})
+@branch(
+  ({
+    health: {
+      status: { good, syncing }
+    }
+  }) => good || syncing,
+  // Only call light.js chainName$ if we're syncing or good
```

Comment:
why? we don't need to be synced/good to query the chainId RPC

Reply:
i just noticed that this approach was used elsewhere in the code when `chainName` was being used

Reply:
I've pushed commit 20e3088f0147701b3c45412ccbf87f6c965c9e27 that addresses this comment